app/presenters/confirmation_letter_presenter.rb

```diff
@@ -15,80 +10,78 @@ def sorted_deregistered_registration_exemptions
     registration_exemptions_with_exemptions.where("state != ?", :active).order_by_state_then_exemption_id
   end
 
-  # Used only if the page contains a single letter.
-  def webpage_title
-    I18n.t("confirmation_letter.single_title", reg_number: registration.reference)
+  def date_of_letter
+    Time.now.in_time_zone("London").to_date.to_formatted_s(:day_month_year)
```

Comment:
I don't think we need this since https://github.com/DEFRA/waste-exemptions-back-office-ta/pull/166